examples/utils/import-testing/src/test/tsconfig.json

```diff
@@ -14,6 +14,9 @@
 		// disable it to prevent that from erroring when combined with "skipLibCheck".
 		// TODO: these errors should be fixed!
 		"exactOptionalPropertyTypes": false,
+		// FixRecursiveArraySchema depends on compilation order dependent behavior in the packages that import the generated d.ts file with libCheck enabled.
+		// Since this package imports such an export, disable incrementality to avoid issues with the build that require clean building in come cases.
+		"incremental": false,
```

Comment:
This seems unrelated and should probably get its own PR

Reply:
This is done because of JsonAsTree.array . This PR actually modifies the d.ts file that its defined in due to the comment fix so its sort of related. Its possible this PR could cause an incremental break without this change.